src/index.html

```diff
@@ -21,7 +24,482 @@
+        </div>
+        <div class="card__price">
+          <p class="card__price-title">Price:</p>
+          <p class="card__price-amount">$2,199</p>
+        </div>
+        <a
+          href="#"
+          class="card__buy"
+          data-qa="card-hover"
+        >
+          BUY
+        </a>
+      </div>
+
+      <div class="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="iMac A1419 on a white background"
+          class="card__img"
+        />
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__review">
+          <div class="stars stars--4">
+            <div
+              class="stars__star"
+              aria-label="star"
+            ></div>
+            <div
+              class="stars__star"
+              aria-label="star"
+            ></div>
+            <div
+              class="stars__star"
+              aria-label="star"
+            ></div>
+            <div
+              class="stars__star"
+              aria-label="star"
+            ></div>
+            <div
+              class="stars__star"
+              aria-label="star"
+            ></div>
+          </div>
+          <p class="card__review-amount">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__price-title">Price:</p>
+          <p class="card__price-amount">$2,199</p>
+        </div>
+        <a
+          href="#"
+          class="card__buy"
+          data-qa="card-hover"
+        >
+          BUY
+        </a>
+      </div>
+
+      <div class="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="iMac A1419 on a white background"
+          class="card__img"
+        />
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__review">
+          <div class="stars stars--4">
+            <div
+              class="stars__star"
+              aria-label="star"
+            ></div>
+            <div
+              class="stars__star"
+              aria-label="star"
+            ></div>
+            <div
+              class="stars__star"
+              aria-label="star"
+            ></div>
+            <div
+              class="stars__star"
+              aria-label="star"
+            ></div>
+            <div
+              class="stars__star"
+              aria-label="star"
+            ></div>
+          </div>
+          <p class="card__review-amount">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__price-title">Price:</p>
+          <p class="card__price-amount">$2,199</p>
+        </div>
+        <a
+          href="#"
+          class="card__buy"
+          data-qa="card-hover"
+        >
+          BUY
+        </a>
+      </div>
```

Comment:
It seems you have repeated the same block of code for the card multiple times. This is not DRY (Don't Repeat Yourself) code. In a real-world scenario, this would be handled dynamically using JavaScript or a back-end language to loop through the data and generate the HTML for each card. If this is just for the purpose of the task and you don't have to use dynamic generation, you should consider using a templating engine that allows you to use loops and conditional logic inside your HTML file, like EJS, Pug, Handlebars, etc.